react/components/BuyButton/index.js

```diff
@@ -1,12 +1,12 @@
-import React, { Component } from 'react'
+import React, { Component, Fragment } from 'react'
 import PropTypes from 'prop-types'
-import { ApolloConsumer } from 'react-apollo'
 import find from 'lodash/find'
-import Button from '@vtex/styleguide/lib/Button'
+import { Button } from 'vtex.styleguide'
 import { injectIntl, intlShape } from 'react-intl'
-
-import ADD_TO_CART_MUTATION from './mutations/addToCartMutation.gql'
-import ORDER_FORM_QUERY from './queries/orderFormQuery.gql'
+import {
+  orderFormConsumer,
+  contextPropTypes,
+} from 'vtex.store/OrderFormContext'
 
 const CONSTANTS = {
   EVENT_SUCCESS: 'item:add',
```

Comment:
remove

Reply:
Actually, it is needed on the toast event.

Reply:
Actually, it is needed to toast the success message